router_test.go

```
@@ -369,31 +373,224 @@ func Test_Router_NotFound_HTML_Inject(t *testing.T) {
+			}
+
+			routes = append(routes, routeMsg)
+		}
+	}
+
+	for _, route := range routes {
+		require.Equal(tb, expectedRoutesCount, strings.Count(route.handlers, " "))
+	}
+}
+
+func verifyThereAreNoRoutes(tb testing.TB, app *App) {
```

golangci-lint (1.62.2)

test helper function should start from tb.Helper()

(thelper)